### PR TITLE
connect-pg-simple: add types

### PR DESCRIPTION
**Pull request by pasieronen**
Please fill in this template.

- [X] Use a meaningful title for the pull request. Include the name of the package modified.
- [X] Test the change in your own code. (Compile and run.)
- [X] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [X] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [X] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [X] The package does not provide its own types, and you can not add them.
- [X] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [X] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [X] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, and `strictNullChecks` set to `true`.

### CODE REVIEW

File: types/connect-pg-simple/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+import { RequestHandler } from "express";
+import { Store, SessionOptions } from "express-session";
+import { Pool, PoolConfig } from "pg";
+
+declare function connectPgSimple(session: (options?: SessionOptions) => RequestHandler): connectPgSimple.PGStore;
+
+declare namespace connectPgSimple {
+  interface PGStore {
+      new (options: PGStoreOptions): Store;
```
Comment (plantain-00) [inline types/connect-pg-simple/index.d.ts]: `options` should be optional. https://github.com/voxpelli/node-connect-pg-simple/blob/master/index.js#L15

File: types/connect-pg-simple/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
+// Type definitions for connect-pg-simple 4.2
+// Project: https://github.com/voxpelli/node-connect-pg-simple#readme
+// Definitions by: Pasi Eronen <https://github.com/pasieronen>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.4
```
Comment (plantain-00) [inline types/connect-pg-simple/index.d.ts]: This can be Typescript 2.5 here.

**Review by plantain-00 — CHANGES_REQUESTED** [review]
There are 2 missing methods, can you add them?https://github.com/voxpelli/node-connect-pg-simple#useful-methods

**Comment by pasieronen** [comment]
@plantain-00, thanks for the review! And good catch about options being optional. 

I also added the close/pruneSessions methods. This was a bit trickier than I thought (I had to change the interface to a class), but at least the tests seem to be happy.

**Review by plantain-00 — APPROVED** [review]

File: types/connect-pg-simple/connect-pg-simple-tests.ts [hunk 3]
```diff
@@ -0,0 +1,44 @@
+import * as connectPgSimple from "connect-pg-simple";
```
Comment (RyanCavanaugh) [inline types/connect-pg-simple/connect-pg-simple-tests.ts]: Classes and functions cannot be imported using `import * as` syntax. Please fix either the test or the definition file to use a correct import form.

See also https://stackoverflow.com/questions/39415661/what-does-resolves-to-a-non-module-entity-and-cannot-be-imported-using-this

**Review by RyanCavanaugh — CHANGES_REQUESTED** [review]

**Comment by pasieronen** [comment]
@RyanCavanaugh, thanks for your review! I've now fixed the tests to use the `import connectPgSimple = require("connect-pg-simple")` form.

**Pull request merged**